vip-jetpack/connection-pilot/class-jetpack-connection-controls.php

```diff
@@ -215,49 +217,22 @@ public static function connect_vaultpress() {
 	/**
 	 * Provision the site with WP.com.
 	 *
-	 * @see https://github.com/Automattic/host-partner-documentation/blob/master/jetpack/plan-provisioning.md
-	 *
 	 * @param int $user_id The VIP machine user ID.
-	 *
-	 * @return mixed bool|\WP_Error True if provisioning worked, \WP_Error otherwise.
+	 * @return mixed bool|WP_Error True if provisioning worked, WP_Error otherwise.
 	 */
-	private static function provision_site( $user_id ) {
-		// TODO: This is a high-risk possible problem on the CLI/CRON containers. Needs testing.
-		$script_path = __DIR__ . '/bin/partner-provision.sh';
-
-		// Can use JP core's script once the changes here are made: https://github.com/Automattic/vip-go-mu-plugins/pull/986.
-		// $script_path = WPMU_PLUGIN_DIR . '/jetpack/bin/partner-provision.sh';
-
-		$cmd = sprintf(
-			'%s --partner_id=%s --partner_secret=%s --url=%s --user=%s --wpcom_user_id=%s --partner-tracking-id=%s --plan="professional" --force_connect=1 --allow-root',
-			$script_path,
-			escapeshellarg( WPCOM_VIP_JP_START_API_CLIENT_ID ),
-			escapeshellarg( WPCOM_VIP_JP_START_API_CLIENT_SECRET ),
-			escapeshellarg( get_site_url() ),
-			escapeshellarg( $user_id ),
-			escapeshellarg( WPCOM_VIP_JP_START_WPCOM_USER_ID ),
-			escapeshellarg( VIP_GO_APP_ID )
-		);
-
-		// phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.system_calls_exec
-		exec( $cmd, $script_output );
-		$script_output_json = json_decode( end( $script_output ) );
-
-		if ( ! $script_output_json ) {
-			return new \WP_Error( 'jp-cxn-pilot-provision-invalid-output', 'Could not parse script output. - ' . $script_output );
-		} elseif ( isset( $script_output_json->error_code ) ) {
-			return new \WP_Error( 'jp-cxn-pilot-provision-error', sprintf( 'Failed to provision site. Error (%s): %s', $script_output_json->error_code, $script_output_json->error_message ) );
-		} elseif ( ! isset( $script_output_json->success ) || true !== $script_output_json->success ) {
-			return new \WP_Error( 'jp-cxn-pilot-provision-error-unknown', 'Failed to provision site. Unknown Error.' );
+	public static function provision_site( $user_id ) {
+		$provision_result = new WP_Error( 'jp-cxn-pilot-provisioning-error', 'Unable to use Provisioner.' );
```

Comment:
Can we put the error inside an `else`, so we don't have to create the object if we're able to provision?